### PR TITLE
Migrate messageBox to webcomponents

### PR DESCRIPTION
**Pull request by akucharska**
**Description**

Changes proposed in this pull request:

- Migrate MessageBox
- Migrate Buttons inside MessageBox

**Related issue(s)**

### CODE REVIEW

File: tests/integration/tests/namespace/test-namespace-overview.spec.js [hunk 1]
```diff
@@ -54,27 +54,32 @@ context(
...
     });
 
     it('Check if limit ranges and resource quota exist', () => {
       cy.contains('b', LIMIT_NAME).should('not.exist');
 
-      cy.contains('b', NEW_LIMIT_NAME).should('not.exist');
+      // cy.contains('b', NEW_LIMIT_NAME).should('not.exist');
 
-      cy.contains('b', QUOTA_NAME).should('not.exist');
+      // cy.contains('b', QUOTA_NAME).should('not.exist');
```
Comment (mrCherry97) [inline tests/integration/tests/namespace/test-namespace-overview.spec.js:82]: why it's commented?

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: src/components/App/IncorrectPath.js [hunk 2]
```diff
@@ -65,20 +64,16 @@ export function IncorrectPath({ to, title = '', message = '' }) {
...
-          compact
-          onClick={() => navigate(to)}
-        >
+        <Button design="Attention" onClick={() => navigate(to)}>
           {t('common.buttons.ok')}
         </Button>,
       ]}
-      show={true}
+      open={true}
+      onClose={() => {}}
```
[the remark this answers is not in the document]
Comment (akucharska) [reply src/components/App/IncorrectPath.js:76]: yes, onClose is required

**Review by mrCherry97 — APPROVED** [review]

**Pull request merged**